docs/src/main/asciidoc/security-guide.adoc

```diff
@@ -61,264 +64,42 @@ public class SubjectExposingResource {
-|quarkus.security.file.enabled|false|Determine whether security via the file realm is enabled
-|quarkus.security.file.auth-mechanism|BASIC|Name of authentication mechanism to use
-|quarkus.security.file.realm-name|Quarkus|Name to assign the security realm
-|quarkus.security.file.users|users.properties|Classpath resource name of properties file containing user to password mappings; see <<Users.properties>>
-|quarkus.security.file.roles|roles.properties|Classpath resource name of properties file containing user to role mappings; see <<Roles.properties>>
-|===
-
-.example application.properties file section for property files realm
-[source,properties]
---
-quarkus.security.file.enabled=true
-quarkus.security.file.users=test-users.properties
-quarkus.security.file.roles=test-roles.properties
-quarkus.security.file.auth-mechanism=BASIC
-quarkus.security.file.realm-name=MyRealm
---
-
-==== Users.properties
-The `quarkus.security.users` configuration property specifies a classpath resource which is a properties file with a user to password mapping, one per line. The following <<test-users-example>> illustrates the format:
-
-[#test-users-example]
-.example test-users.properties file
-[source,properties]
---
-scott=jb0ss <1>
-jdoe=p4ssw0rd <2>
-stuart=test
-noadmin=n0Adm1n
---
-<1> User `scott` has password defined as `jb0ss`
-<2> User `jdoe` has password defined as `p4ssw0rd`
+|Extension |Description
 
+|link:elytron-properties-guide.html[quarkus-elytron-security-properties-file]
+|Provides support for simples properties files that can be used for testing security. This supports both embedding user info in `application.properties` and standalone properties files.
 
-==== Roles.properties
+|link:oauth2-guide.html[quarkus-elytron-security-oath2]
+|Provides support for OAth2 flows using Elytron. This extension will likely be deprecated soon and replaced by a reactive Vert.x version.
 
-.example test-roles.properties file
-[source,properties]
---
-scott=Admin,admin,Tester,user <1>
-jdoe=NoRolesUser <2>
-stuart=admin,user <3>
-noadmin=user
---
-<1> User `scott` has been assigned the roles `Admin`, `admin`, `Tester` and `user`
-<2> User `jdoe` has been assigned the role `NoRolesUser`
-<3> User `stuart` has been assigned the roles `admin` and `user`.
-
-Given these role mappings, only user `scott` would be allowed to access the `/subject/secured` endpoint from the <<subject-example>>.
+|link:jwt-guide.html[quarkus-smallrye-jwt]
+|A Microprofile JWT implementation that provides support for authenticating use Json Web Tokens. This also allows you to inject the token and claims into the application as per the MP JWT spec.
 
-### Embedded Realm Configuration
-The embedded realm also supports mapping of users to password and users to roles. It uses the main application.properties Quarkus configuration file to embed this information. To enable and configure it, the following configuration properties are used:
+|link:keycloak-guide.html[quarkus-vertx-keycloak]
```

Comment:
why did we rename it `vertx-keycloak`? It doesn't work at all in a servlet environment?

Reply:
Because our naming format is productname-feature. In this case vertx is providing Keycloak, so according to our standard naming rules this should be the name. I am happy to change it to just keycloak, but it might even need to be something more generic (as it may just be the open id connect implementation)

Reply:
Can we have a discussion about the naming on the mailing list? In respect to the `elytron` thing and `vertx-keycloak`? Let's not block this PR for that.